toolboxTreppe/toolboxTreppe.ino

```diff
@@ -43,40 +42,37 @@ void setup() {
   pinMode(buttonPin3, INPUT_PULLUP);
   pinMode(buttonPin4, INPUT_PULLUP);
   Serial.begin(9600);
-  Serial.println("Hello World!");
+  debugPrint("Hello World!");
 }
 
 //----------------------------------LOOP -------------------
 void loop() 
 { 
-  static uint8_t y;
-  static uint8_t x;
-  static uint8_t gegenstrecke = kMatrixHeight-1;
-  
   //Wenn Schalter oben ist, starte Animation.
   if (digitalRead(buttonPin1) == HIGH) {
-    //Wenn der Schalter in der mitte ist, aktiviere debugfunktion
-    if (digitalRead(buttonPin2) == HIGH) {
-      debug(y, x, gegenstrecke);
-    }
     animation1(y, x, gegenstrecke);
   } 
   //LEDs ausschalten, wenn der switch unten ist (Pinkontakt 2+3)
-  else if ((digitalRead(buttonPin1) == LOW) && (digitalRead(buttonPin2) == HIGH)) {
+  else if (digitalRead(buttonPin2) == HIGH) {
     powerOff();
   }
 }
 
-
-
-
 //-------------------------------Animationsfunktion ------------
 void animation1(uint8_t& y, uint8_t& x, uint8_t& gegenstrecke) {
   static uint8_t xA;
   static uint8_t xB;
   static unsigned long xMillis;
   static long onewayMillis;
   static uint8_t hue;
+  static uint8_t y;
+  static uint8_t x;
+  static uint8_t gegenstrecke = kMatrixHeight-1;
+  debugPrint(
+    "x: " + x + "   "
+    + "y: " + y + "   "
+    + "gegenstrecke: " + gegenstrecke
```

Comment:
like this you read them as chars which will not work out well and is not readable.